package.json

```diff
@@ -7,37 +7,26 @@
 		"Chris Trevino <chtrevin@microsoft.com>"
 	],
 	"scripts": {
-		"clean": "lerna run clean --stream",
+		"clean": "lerna run clean --stream --parallel",
 		"build": "lerna run build --stream",
 		"test": "lerna run test --stream",
 		"publish_packages": "lerna publish",
+		"essex": "node ./packages/scripts/lib/index.js",
 		"release": "run-s build publish_packages",
-		"lint": "node ./packages/scripts/lib/index.js lint --docs --strict",
-		"prettify": "node ./packages/scripts/lib/index.js prettify",
-		"audit_dependencies": "node ./packages/scripts/lib/index.js audit",
-		"git_is_clean": "node ./packages/scripts/lib/index.js git-is-clean",
+		"lint": "yarn essex lint --docs --strict",
+		"prettify": "yarn essex prettify",
+		"audit_dependencies": "yarn essex audit",
+		"git_is_clean": "yarn essex git-is-clean",
 		"ci": "run-s build lint test git_is_clean"
 	},
 	"workspaces": {
 		"packages": [
 			"packages/*"
 		]
 	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "node ./packages/scripts/lib/index.js pre-commit",
-			"commit-msg": "node ./packages/scripts/lib/index.js commit-msg"
-		}
-	},
 	"devDependencies": {
-		"@essex/scripts": "^7.0.3",
-		"@typescript-eslint/eslint-plugin": "^3.0.0",
-		"@typescript-eslint/parser": "^3.0.0",
-		"crlf": "^1.1.1",
```

Comment:
We're not using the ones you've removed elsewhere?

Reply:
With Yarn v2, resolving the @typescript-eslint/x packages actually works, so there's no need to include them as peers here. I pushed CRLF down into the scripts package - Yarn v2 generally works better if binary deps that packages depend on are declared where they're used

Reply:
OK, sounds good.  I just wanted to make sure, for example that the `crlf` dep got moved over to the packages that were dev dependent on it.